petab/yaml.py

```diff
@@ -253,16 +255,24 @@ def get_rel_to_yaml(paths: Union[List[str], None]):
         measurement_files = get_rel_to_yaml(measurement_files)
         observable_files = get_rel_to_yaml(observable_files)
         visualization_files = get_rel_to_yaml(visualization_files)
-
         parameter_file = get_rel_to_yaml([parameter_file])[0]
+        mapping_file = get_rel_to_yaml([mapping_file])[0]\
+            if mapping_file else None
```

Comment:
```suggestion
        mapping_file = get_rel_to_yaml(mapping_file)
```

Reply:
Ah, now I'm confused by my suggestion, because `get_rel_to_yaml` expects `List[str]`...

Reply:
And it makes more sense to change it to a list of filenames for coherence with the other file types.